wire-runtime-swift/src/main/swift/ProtoCodable/ProtoEncoder.swift

```diff
@@ -80,7 +80,9 @@ public final class ProtoEncoder {
         // Use the size of the struct as an initial estimate for the space needed.
         let structSize = MemoryLayout.size(ofValue: value)
 
-        let writer = ProtoWriter(data: .init(capacity: structSize))
+        let writer = ProtoWriter(data: .init(capacity: structSize),
+                                 outputFormatting: [],
+                                 protoSyntax: T.self.protoSyntax ?? .proto2)
```

Comment:
I wonder why we made this optional...can't remember now. This is right though, as "no syntax definition" means proto2.